### DISCUSSION TITLE
How to get Public Sponsor badge?

### DISCUSSION THREAD

**Discussion by hpelkonen** [community/community#30216] [Other Feature Feedback, Questions, & Ideas] [answered]
I saw many github profiles have "Public Sponsor" badge, what is it and how to get it?

**Comment by codestheos** [comment, marked as answer]
You can sponsor contributors, receive updates on developers and organizations you sponsor, and display a sponsor badge throughout GitHub. Plz check more details the way to get the github profile badges [here](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by Danyalkasiri** [reply]
``
